docs/integrations/containers-orchestration/kubernetes-control-plane.md

```diff
@@ -111,29 +111,33 @@ This section provides instructions for installing the Kubernetes Control Plane A
 
 All the dashboards are linked to the Explore tab and can be easily accessed by clicking the Cluster, in the left navigation pane of Explore tab.
 
-
 ### Step 1: Determine custom data filters for source categories
 
-When you install the FluentD plugin and Prometheus, you supply custom data filters that match the source categories that the FluentD plugin generated for your Kubernetes logs and metrics. The plugin generates the source categories dynamically, and they can vary by environment.
+When you installed the [Sumo Logic Kubernetes Collection Helm Chart](https://github.com/SumoLogic/sumologic-kubernetes-collection) in your cluster,
+logs are collected by [Sumo Logic Distribution for OpenTelemetry Collector](https://github.com/SumoLogic/sumologic-otel-collector).
+Based on the configuration some information are extracted as metadata added to logs, one of the special metadata is `Source Category` which helps to search data.
```

Comment:
```suggestion
We add several built-in metadata fields to all ingested log records. We're going to use the `Source Category` field for querying in the following steps.
```

Reply:
This proposition does not match the style. It will be the only sentence with "we.

Reply:
@kimsauce Could you help here?

Reply:
I agree @kkujawa-sumo . Using "we" would make it inconsistent with the voice of this particular doc.

Reply:
How about
```suggestion
All ingested log records will have certain built-in metadata fields set. One of them is the `Source Category` field, which can be used to specify custom data filters for this app.
```

Reply:
Changed to @swiatekm-sumo's  proposition.